github/ci/prow/templates/mirror-config.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```

Comment:
Personally, I'd prefer to split the addition of the mirror service from the addition of the new prow jobs into different commits/PRs - but not that important in this stage.

Reply:
It's the infra for optional new jobs, so I don't see much of difference and it kind of belongs together. I agree that there are situations where it is better to split that into separte PRs. The mirror is in a separate commit.